test/integration/concrete/lockup/withdraw/withdraw.tree

```diff
@@ -8,66 +8,66 @@ withdraw.t.sol
        ├── given the stream's status is "DEPLETED"
        │  └── it should revert
        └── given the stream's status is not "DEPLETED"
-          ├── when the caller is unauthorized
-          │  ├── when the caller is the sender
-          │  │  └── it should revert
-          │  ├── when the caller is a malicious third party
-          │  │  └── it should revert
-          │  └── when the caller is a former recipient
-          │     └── it should revert
-          └── when the caller is authorized
```

Comment:
> these branches should remained

If we keep this branch, what would it mean to say "authorized" now since every address is authorized to withdraw? I tried to think about a keyword that can be used to distinguish users such as sender, recipient, approved third party from rest of the addresses but couldn't come up with one. 

> we could say "an unknown address" instead of "malicious third party"

To use where?

Reply:
agree that "authorized" is not the best wording

the idea is that anyone is _allowed_ to call it but not everyone is authorized to withdraw to anyone

Reply:
are you ok to leave the "authorized" word until we find a better one?

Reply:
Yeah I am OK with this.